cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -245,7 +245,7 @@
 default['travis_build_environment']['neo4j_version'] = '3.2.1'
 default['travis_build_environment']['neo4j_checksum'] = '24fd6a704e0d80c4b4f9a3d17ce0db23f258a8cdcfa1eb28d7803b7d1811ee96'
 
-default['travis_build_environment']['mercurial_install_type'] = 'ppa'
+default['travis_build_environment']['mercurial_install_type'] = 'src'
 if node['kernel']['machine'] == 'ppc64le'
```

Comment:
I think we can also drop this condition, since it doesn't change anything anymore.